homeassistant/components/aquacell/config_flow.py

```diff
@@ -30,10 +36,31 @@ class AquaCellConfigFlow(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
+    _brand: Brand = Brand.AQUACELL
+
     async def async_step_user(
         self, user_input: dict[str, Any] | None = None
     ) -> ConfigFlowResult:
-        """Handle the initial step."""
+        """Handle the initial step via config flow."""
+        if user_input:
+            self._brand = user_input[CONF_BRAND]
+            return await self.async_step_cloud()
+
+        return self.async_show_form(
+            step_id="user",
+            data_schema=vol.Schema(
+                {
+                    vol.Required(CONF_BRAND, default=self._brand): vol.In(
+                        {key: brand.name for key, brand in SUPPORTED_BRANDS.items()}
+                    ),
+                }
+            ),
+        )
+
+    async def async_step_cloud(
```

Comment:
Just a thought, we could just combine these steps into one, because the schema of the second step doesn't change

Reply:
Yeah that does make it a bit less complex. I've made the change.